dask_sql/mappings.py

```diff
@@ -168,6 +174,8 @@ def sql_to_python_value(sql_type: "SqlTypeName", literal_value: Any) -> Any:
     elif (
         sql_type == SqlTypeName.TIMESTAMP
         or sql_type == SqlTypeName.TIME
+        or sql_type == SqlTypeName.TIME32
+        or sql_type == SqlTypeName.TIME64
```

Comment:
I mostly just added this to match the existing logic - the pytest still functions without it.